scripts/bbl-info.js

```diff
@@ -4,10 +4,10 @@ const fetch = require('node-fetch');
 
 const boroughArray = [null, 'Manhattan', 'The Bronx', 'Brooklyn', 'Queens', 'Staten Island'];
 
-const bblRegex = /\b[1-5]{1}[0-9]{5}[0-9]{4}\b/;
+const bblRegex = /^bbl \b[1-5]{1}[0-9]{5}[0-9]{4}\b/;
 
 module.exports = (robot) => {
-  robot.hear(bblRegex, async (res) => {
+  robot.response(bblRegex, async (res) => {
```

Comment:
use `respond` here, for both bbl search and geosupport. This requires a user actually `@plannerbot` to get a response, which I think is a better experience. This also removes the tendency of plannerbot to read BBLs in random links.